docker-entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+# Add elasticsearch as command if needed
+if [[ "$1" == "-"* ]]; then
+  set -- elasticsearch "$@"
+fi
+
+# Drop root privileges if we are running elasticsearch
+if [[ "$1" == "elasticsearch" ]]; then
```

Comment:
I'm sure @tianon would point this out, so I'll do it first: no need for the magic of the `[[`, the single `[` will work here (and unnecessary use of double quotes, and only a single `=` is needed).
```bash
if [ "$1" = 'elasticsearch' ]; then
```